core/src/main/scala/time.scala

```diff
@@ -15,8 +15,13 @@ object time:
   extension (d: LocalDate) def adjust(a: TemporalAdjuster): LocalDate = d.`with`(a)
 
   extension (d: LocalDateTime)
+    /** Seconds from epoch.
+      *
+      * This uses a checked conversion, it will overflow and raise if the year is after 2038.
+      */
+    def toSeconds: Int = Math.toIntExact(d.toEpochSecond(utc_))
```

Comment:
We can use `LocalDateTime`, `Instant` to present sometime in the future, so this is not actually safe? I guess to make things more consistent, we should return Long here instead of return Int everywhere else.

Reply:
Yes it's not totally safe because even in 2024 we could theoretically construct a LocalDateTime or Instant that corresponds to year 2038. That's why it's a checked cast for the LocalDateTime and Instant extension methods but unchecked for `nowSeconds`

It's a little unclean but working with ints when ints suffice is worth it, for now imo. This won't come up as an issue in lila until 2036 or later.

Reply:
I think I just recently changed the other method to Int, yes we could undo them and return long. But do we really need to?  Idk